include/swift/Option/FrontendOptions.td

```diff
@@ -317,6 +317,10 @@ def enable_resilience : Flag<["-"], "enable-resilience">,
 def enable_experimental_async_top_level :
   Flag<["-"], "enable-experimental-async-top-level">,
   HelpText<"Enable experimental concurrency in top-level code">;
+
+def enable_experimental_layout_prespecialization :
```

Comment:
Instead of adding this new flag, could you introduce this as an `EXPERIMENTAL_FEATURE` in `Features.def`?

Reply:
Ah, good catch. I looked at other flags and must have gotten one that uses both and replicated that. I removed the extra flag and replaced usages with the `-enable-experimental-feature` flag in https://github.com/apple/swift/pull/61451.